Applications/DataExplorer/DataView/NetCdfConfigureDialog.cpp

```diff
@@ -43,7 +41,7 @@ NetCdfConfigureDialog::~NetCdfConfigureDialog()
 void NetCdfConfigureDialog::accept()
 {
 	QMessageBox valueErrorBox;
-	if (_currentVar->num_dims() < 3){
+	if (_currentVar->num_dims() < 2){
```

Comment:
NetCDF can store multiple variables of varying dimensions, e.g. x=3 has dim=0, but I can also store time- or space ranges (dim = 1), rasters (dim=2), rasters over time or 3-d rasters (dim=3), etc. If you have a netcdf file containing climate data you can easily have 100 or 200 variables.
These variables are interconnected, e.g. a geo-referenced raster will have its extend in x and y are stored as arrays (so you need not necessarily have equidistant or isotropic pixels).
_currentVar is the variable that is currently selected in the dialogue and which all other input in the dialogue depends on. We ignore selectable variables with dim<2 because we cannot convert those into raster- or mesh objects.